### ISSUE TITLE
Bug in pagination: When scroll download to get next page, always get the gists in page=2

### ISSUE THREAD

**Issue by huangxiaobin328** [cmoulton/grokSwiftREST_v1.1#3]
```
let result: (path: String, parameters: [String: AnyObject]?) = {
  switch self {
case .GetPublic:
  return ("/gists/public", nil)
case .GetAtPath(let path):
  let URL = NSURL(string: path)
  let relativePath = URL!.relativePath! // in this code return "/gists/public", NOT "/gists/public?page=nextId"
  return (relativePath, nil) 
  }
}()
```

**Comment by cmoulton** [comment]
This was fixed in 72ae9f2923127bb7c8a2ad1661cc95118f0f7e94

```
case .GetAtPath(let path):
    // already have the full URL, so just return it
    return NSURL(string: path)!
}
```

**Comment by huangxiaobin328** [comment]
Thank you for your great book.
